### PR TITLE
feat(chaos): migrate await-message-correlation.sh to Kotlin

### PR DESCRIPTION
**Pull request by pihme**
migrate await-message-correlation.sh to Kotlin

### CODE REVIEW

**Comment by pihme** [comment]
I was hoping I could migrate a bunch in one go, but many of the other workers are a bit trickier. So it just stays this one :-(

File: core/chaos-workers/chaos-worker/src/main/kotlin/io/zeebe/chaos/chaosMain.kt [hunk 1]
```diff
@@ -55,13 +55,16 @@ fun main() {
...
+        .filterNot { it.equals(AwaitMessageCorrelationHandler.JOB_TYPE) }
         .filterNot { it.equals(AwaitProcessWithResultHandler.JOB_TYPE) }
         .filterNot { it.equals(DeployMultipleVersionsHandler.JOB_TYPE) }
         .forEach { script ->
             LOG.info("Start worker with type `$script`")
             zeebeClient.newWorker().jobType(script).handler(::handler).open()
         }
 
+    zeebeClient.newWorker().jobType(AwaitMessageCorrelationHandler.JOB_TYPE)
+        .handler(AwaitProcessWithResultHandler()).open()
```
Comment (Zelldon) [inline core/chaos-workers/chaos-worker/src/main/kotlin/io/zeebe/chaos/chaosMain.kt:67]: I assume you want to call the other one :) 

```suggestion
        .handler(AwaitMessageCorrelationHandler()).open()
```

**Review by Zelldon — APPROVED** [review]
Awesome thanks @pihme you know "Kleinvieh macht auch mist :)"

:rocket:

**Pull request merged**